target/linux/mediatek/dts/mt7986b-mercusys-mr90x-v1.dts

```diff
@@ -41,32 +42,42 @@
 		compatible = "gpio-leds";
 
 	 	led-0 {
-			label = "green:lan2";
+			color = <LED_COLOR_ID_GREEN>;
+			function = LED_FUNCTION_LAN;
+			function-enumerator = <2>;
 			gpios = <&pio 7 GPIO_ACTIVE_LOW>;
 		};
 
 		led-1 {
-			label = "green:lan1";
+			color = <LED_COLOR_ID_GREEN>;
+			function = LED_FUNCTION_LAN;
+			function-enumerator = <1>;
 			gpios = <&pio 9 GPIO_ACTIVE_LOW>;
 		};
 
 		led-2 {
-			label = "green:lan0";
+			color = <LED_COLOR_ID_GREEN>;
+			function = LED_FUNCTION_LAN;
+			function-enumerator = <0>;
 			gpios = <&pio 12 GPIO_ACTIVE_LOW>;
 		};
 
 		led-3 {
-			label = "green:wan";
+			color = <LED_COLOR_ID_GREEN>;
+			function = LED_FUNCTION_WAN;
 			gpios = <&pio 13 GPIO_ACTIVE_LOW>;
 		};
 
 		led-4 {
-			label = "orange:status";
+			color = <LED_COLOR_ID_AMBER>;
+			function = LED_FUNCTION_STATUS;
 			gpios = <&pio 16 GPIO_ACTIVE_HIGH>;
 		};
 
 		led_status_green: led-5 {
 			label = "green:status";
```

Comment:
Perhaps wait for #13837 so you could drop the label here too?

Reply:
Yes, that probably makes sense.

Reply:
Let's do this after https://github.com/openwrt/openwrt/pull/13837 is merged.

Reply:
It has just been merged :)